README.md

```diff
@@ -4595,6 +4596,61 @@ It is mandatory to define the data type in dataclass. However, If you don't want
 
 [_Return to the Top_](#python-cheatsheet)
 
+```python
+>>> from dataclasses import dataclass
+>>> from typing import Any
+
+>>> @dataclass
+... class WithoutExplicitTypes:
+...    name: Any
+...    value: Any = 42
+...
+```
```

Comment:
This looks to be duplicated from the section above?

Reply:
I agree. This is my failure.